### PR TITLE
NHS.UK prototype re-design

### PR DESCRIPTION
**Pull request by craig-shenton**
Re-design following Digital Service Manual principles and SEO improvements .

### CODE REVIEW

**Review by maryamanuelnhsx — CHANGES_REQUESTED** [review]
Hi @craig-shenton. this is absolutely **OUTSTANDING!**. I can't thank you enough for your hard work in re-designing this website to make it in line with NHS front end design principles and having a think about SEO.  Pycom website is **much more** functional now, thank you. 

There are some minor changes to be made before merging to main, mainly around the code of conduct doc, contents page reordering and minor typos. Then it should be good to go ask v2. There are also a few thing I'd like to discuss in person but we can do that on wednesday. 

Thanks again, brilliant work as always. This needs to be shared with the AU at some point :)

File: python-resources.html [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+            </div>
+          </div>
+        </li>
+        <li class="nhsuk-grid-column-one-half nhsuk-card-group__item">
+          <div class="nhsuk-card nhsuk-card--clickable">
+            <div class="nhsuk-card__content">
+              <h2 class="nhsuk-card__heading nhsuk-heading-m">
+                <a class="nhsuk-card__link" href="https://github.com/pamoroso/free-python-books">Free Python Books</a>
+              </h2>
+              <p class="nhsuk-card__description">A list of Python books in English that are free to read online or download..</p>
```
[the remark this answers is not in the document]
Comment (maryamanuelnhsx) [reply python-resources.html:229]: > @craig-shenton - super minor edit, remove the double full stop.

@craig-shenton next to "online and download"

File: code-of-conduct.html [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+      <ul class="nhsuk-footer__list">
+        <li class="nhsuk-footer__list-item">Developed by <a class="nhsuk-footer__list-item-link" href="https://www.nhsx.nhs.uk/">NHSX</a></li>
+        <li class="nhsuk-footer__list-item">View the Project on <a class="nhsuk-footer__list-item-link" href="{{ site.github.repository_url }}">GitHub</a></li>
+      </ul>
+      
+      <p class="nhsuk-footer__copyright">&copy; MIT Licence</p>
+    </div>
+  </div>
+</footer>
+
```
[the remark this answers is not in the document]
Comment (maryamanuelnhsx) [reply code-of-conduct.html:260]: @Arouba will update this.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+                <svg class="nhsuk-icon nhsuk-icon__chevron-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
+                  <path d="M15.5 12a1 1 0 0 1-.29.71l-5 5a1 1 0 0 1-1.42-1.42l4.3-4.29-4.3-4.29a1 1 0 0 1 1.42-1.42l5 5a1 1 0 0 1 .29.71z"></path>
+                </svg>
+              </a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+  
+      </header>
```
[the remark this answers is not in the document]
Comment (craig-shenton) [reply index.html:137]: Home was hidden except on mobile, now shows on desktop view

[the remark this answers is not in the document]
Comment (maryamanuelnhsx) [reply index.html]: looks great @craig-shenton

**Comment by craig-shenton** [comment]
Footer on all pages need to copy index.html

File: code-of-conduct.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (Arouba) [reply code-of-conduct.html]: Updated

**Pull request merged**